.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '*'
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Android
+      env:
+        GPG_KEYSTORE: ${{ secrets.GPG_KEYSTORE }}
```

Comment:
You're correctly using secrets here, but you've pushed your keystore in this PR. You should at least replace this commit with another one without the keystore.

But also anyone who can find the old commit hash will be able to download the leaked keystore file.

Reply:
keystore isn't leaked. It's GPG encrypted and the key is stored in secrets.
This practice is actually present in the GitHub documentation: https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#limits-for-secrets

Reply:
Ah, I've never used GPG encryption for anything before, I've only heard about it in the context of encrypting mails.

Reply:
But also, do you then still need to have it committed?

Reply:
Yeah, GitHub says to do so in their official documentation, so it should be pretty safe. See point 4 in the link above.

Reply:
I just noticed, you're using secrets.GPG_KEYSTORE as keystore password and KEYSTORE_PASS as key password.

If they're the same, you can leave out the key password parameter. I would change the secrets like this:
GPG_KEYSTORE -> KEYSTORE_PASS
KEYSTORE_PASS -> KEY_PASS